### ISSUE TITLE
[plugin_manager] Multiple issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#353]
```
Token: fc42b79df8ecfb85af09677b06ef5646 (Don't modify this token!)
Version: 4.1 (04-07-2012)
PHP: 5.3.14
RCMAIL: 0.8-rc
Database: pgsql
SERVER: nginx/0.7.67
----
I.  Issue Description:
Issue 1: Plugin manager doesn't deal with domain less login.
I had to change plugin_manager.php to read that part:
else{
        $temparr = explode('@', $_SESSION['username']);
        $this->domain = strtolower($temparr[1]);
        global $_SERVER;
        if ($this->domain == "") $this->domain = $_SERVER["HTTP_HOST"];
      }

Issue 2: Plugin manager doesn't show the "I agree | I disagree" link in the 
update/warning page.
I had to change the visibility from "display:none" to "display:show" in the 
appropriate div box with Firebug so be able to see the link to click.

Issue 3: Plugin manager doesn't allow me to uncheck google analytics plugin.
In the plugin manager, I'm seeing the google analytics plugin, but I can't 
uncheck it. (If I do so, it uncheck, write "saved config", then reload and it's 
checked again).
I haven't found a way to make it work, except by deleting the folder from the 
plugins.
```

Original issue reported on code.google.com by `boite.po...@gmail.com` on 5 Jul 2012 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
#1 and #2 will be fixed (both issues result from domainless login).

#3 is most likely a configuration issue. Please post the section of 
google_analytics config. Here is the default:

{{{
    'google_analytics' => array( 
      'label_name' => 'google_analytics.pluginname',
      'label_description' => 'google_analytics.plugindescription',
      'protected' => array('mail4us.net'),
      'active' => true
    ),
}}}
```

Original comment by `rol...@liebl.ath.cx` on 6 Jul 2012 at 4:53
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I've deleted the folder to remove the plugin, so I don't have the plugin 
anymore.

I don't what it's worth, but in the initial "update" of the plugin, I've 
downloaded only few of the huge list, yet my logs are cluttered by "
roundcube: PHP Error: Failed to load plugin file 
/path/to/rc/plugins/automatic_addressbook/automatic_addressbook.php" and many 
other (in fact, all the plugins I haven't checked initially.
```

Original comment by `boite.po...@gmail.com` on 6 Jul 2012 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
#1 - comment #1 should be fixed in plugin_manager v4.2.

#2 - comment #2 configuration issue: Remove all plugins you don't have 
installed from plugin_manager config.

Re-open ticket if neccessary.
```

Original comment by `rol...@liebl.ath.cx` on 6 Jul 2012 at 7:52
- Changed state: **Fixed**
